### DISCUSSION TITLE
Flow-pressure relationship

### DISCUSSION THREAD

**Discussion by amayboudellal** [e2nIEE/pandapipes#432] [Q&A] [answered]
Hello,

I'm solving a pipeflow in a district heating network based on historical smart meter data and creating a future scenario where I increase the flow at each sink. This results are an overall increase in flow through the whole system as planned. However, I'm also experiencing a lower pressure in pipes and at junctions where I would actually expect an increase based on the pressure-flow relationship.

Do you have any insights regarding how this could happen?

**Comment by dlohmeier** [comment, marked as answer]
Hello @amayboudellal,
based on your description, it's a little hard to judge, since you only describe behavior that perfectly fits the usual expectations (higher flow to sinks -> lower pressure). Can you give some more details on the flow situation in your network? Do you make any changes to the temperature settings? (Please keep in mind that the hydraulics are calculated prior to the thermal situation based on the input of temperatures that you provide.) What kinds of pressure definition do you give to the model (pumps, external_grids, ... )? How do you generate the data for the boundary conditions regarding the pressure (e.g. do you have historical data that you hand over to your external grid instances)?

**Reply by amayboudellal** [reply]
Thanks for your answer. I just realized I was using an unconstrained external grid that therefore just supplied a different pressure than I originally expected.
